docs/src/userguide/numpy_tutorial.rst

```diff
@@ -472,21 +367,16 @@ here has gone through a lot of thinking and planning already; still they may
 or may not happen depending on available developer time and resources for
 Cython.
 
-1. Support for efficient access to structs/records stored in arrays; currently
-   only primitive types are allowed.
-2. Support for efficient access to complex floating point types in arrays. The
+1. Support for efficient access to complex floating point types in arrays. The
    main obstacle here is getting support for efficient complex datatypes in
    Cython.
```

Comment:
Cython _has_ support for different `complex` types, so I'm not sure if this doesn't work out of the box.